### DISCUSSION TITLE
mac & linux desktop app

### DISCUSSION THREAD

**Discussion by dumorando** [cytrinc/bark-coding#171] [New Features (not block requests)]
yeah

**Comment by raynepaws** [comment]
agreed, i have a linux so i can't see the desktop app

**Comment by mariocraft987** [comment]
yes

**Comment by dumorando** [comment]
> agreed, i have a linux so i can't see the desktop app

same

**Comment by raynepaws** [comment]
i used a windows at school and downloaded the app onto that, and it seemed pretty legit

**Comment by mariocraft987** [comment]
same

**Comment by raynepaws** [comment]
it's probably really easy, we can just use electron or something

**Comment by mariocraft987** [comment]
Yea

**Comment by rubiidev18alt** [comment]
I kinda made 0.1.0 already but yeah, linux users unite! (I'm not one)

**Comment by nmsderp** [comment]
i don't have linux or mac lol (i would never use mac anyway)

**Comment by Bashamega** [comment]
I think we don't need to worry about that.
We could use this cool project, which transforms anything into a app that can be used on Linux or Windows or Mac: 
https://github.com/nativefier/nativefier

If you like the project i can run the command upload the files and also create a downloads page for it
